stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/search/SearchResultListView.kt

```diff
@@ -0,0 +1,75 @@
+package io.getstream.chat.android.ui.search
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.databinding.StreamSearchResultListViewBinding
+
+public class SearchResultListView : LinearLayout {
+
+    private val binding = StreamSearchResultListViewBinding.inflate(LayoutInflater.from(context), this)
+
+    public constructor(context: Context) : super(context) {
+        init(null)
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        init(attrs)
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
+        context,
+        attrs,
+        defStyleAttr
+    ) {
+        init(attrs)
+    }
+
+    private val adapter = SearchResultListAdapter()
+
+    private fun init(attrs: AttributeSet?) {
+        parseAttrs(attrs)
+
+        orientation = VERTICAL
+
+        binding.searchListView.apply {
+            setHasFixedSize(true)
+            adapter = this@SearchResultListView.adapter
+            layoutManager = LinearLayoutManager(context).apply {
```

Comment:
Just out of curiosity - why not setting layout manager and orientation in XML?

Reply:
Oops, this was leftover code from when the entire View itself was a `RecyclerView`, and XML config wasn't available on the `merge` tag 😄 . Moved to XML.